src/tests/unit/cpu/ngraph_transformations/fake_quantize_tokenization_test.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "transformations/op_conversions/fq_decomposition.hpp"
+#include "snippets/pass/collapse_subgraph.hpp"
+#include "fake_quantize_function.hpp"
+#include "snippets/op/subgraph.hpp"
+#include "ngraph_transformations/snippets_mark_skipped.hpp"
+#include "function_helper.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class FakeQuantizeTokenizationTest : public TransformationTestsF {
```

Comment:
This test is very similar to FakeQuantizeDecompositionTest, so it may be a good idea to derive from it to avoid copy-paste. Although there might be some restrictions, because they are in different binaries.